src/main/java/seedu/address/logic/commands/tag/EditCommand.java

```diff
@@ -25,7 +25,7 @@
             + "Note that all contacts or sales associated with this tag "
             + "will be updated automatically with the updated tag.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
```

Comment:
ps still got the parameter here that needs to be updated as well

Reply:
Right sorry haha